analyzers/its/sources/Net5/Net5/S2292.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Net5
+{
+    public class S2292
+    {
+        public record Record
+        {
+            private string field;
+            public string PropWithGetAndInit
```

Comment:
_Nitpick_: Missing EOL
```suggestion
            private string field;
            
            public string PropWithGetAndInit
```